plugins/toolkit/jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererProjectCodeScanTest.kt

```diff
@@ -378,10 +391,5 @@ class CodeWhispererProjectCodeScanTest : CodeWhispererCodeScanTestBase(PythonCod
         ).virtualFile
         totalSize += testYaml.length
         totalLines += testYaml.toNioPath().toFile().readLines().size
-
-        // Adding gitignore file and gitignore file member for testing.
-        // The tests include the markdown file but not these two files.
-        projectRule.fixture.addFileToProject("/.gitignore", "node_modules\n.idea\n.vscode\n.DS_Store").virtualFile
-        projectRule.fixture.addFileToProject("test.idea", "ref: refs/heads/main")
```

Comment:
need to figure out tests

Reply:
Ack. i have been working to write tests including gitignore files.